packages/chain-helpers/src/blockchainApiConnection/TypeRegistry.ts

```diff
@@ -0,0 +1,8 @@
+import { types8 as KILT_TYPES } from '@kiltprotocol/type-definitions'
+import { TypeRegistry } from '@polkadot/types'
```

Comment:
```suggestion
import type { TypeRegistry } from '@polkadot/types'
```

Reply:
actually this is imported by value, not by type. You see below that we instantiate TypeRegistry